### PR TITLE
add Periodic Batching config options

### PR DESCRIPTION
**Pull request by monishkadas-ms**
Updated the logger configuration to accept Periodic Batching Configuration parameters from appsettings.json

### CODE REVIEW

**Comment by tanmaya-panda1** [comment]
@monishkadas-ms Please do update the docs as well

File: README.md [hunk 1]
```diff
@@ -52,9 +52,11 @@ For example:
 new SinkColumnMapping { ColumnName ="Exception", ColumnType ="string", ValuePath = "$.ExceptionEx" }
 ```
```
Comment (ag-ramachandran) [inline README.md:53]: Library versions have to be bumped up

**Review by ag-ramachandran — CHANGES_REQUESTED** [review]
Bump up versions

File: README.md (hunk 1 above)
Comment (monishkadas-ms) [reply README.md:53]: Done!

**Review by ag-ramachandran — APPROVED** [review]

**Review by tanmaya-panda1 — APPROVED** [review]

**Pull request merged**